.env.example

```
@@ -33,3 +33,8 @@ HUMANITY_API_KEY="your-humanity-protocol-api-key"
 HUMANITY_API_URL="https://testnet-api.humanity.org/v1/human/verify"
 # Portal URL for user verification
 NEXT_PUBLIC_HUMANITY_PORTAL_URL="https://testnet.humanity.org"
+
+# === UPLOADTHING ===
+# Get your API keys from https://uploadthing.com/dashboard
+UPLOADTHING_SECRET="sk_live_..."
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)